### DISCUSSION TITLE
Deta 6914ha- series 2 fan/ light controller

### DISCUSSION THREAD

**Discussion by baskarz** [arendst/Tasmota#17874] [Q&A]
Hi all, long time user of tasmota, absolutely love it!
Just purchased a Deta 6914ha series 2 ceiling fan/ light wall controller.
I checked on the tasmota compatible list and the 6914ha is listed as compatible. 
Upon opening it up to flash with tasmota, it would appear this is a newer version and does not contain an esp type chip, instead it has the WB3S chip now. I guess that's what the "series 2" is now.

I see people have replaced the wb3s chip with esp12s and appears to be pin compatible. 
My question is, if I do a chip swap to esp12s, is it just a matter of flashing with tasmota and configuring as per the 6914ha template?

Has anyone done this and had success?
Is the esp12s a true drop in replacement for the wb3s?

Keen to see others experiences with this.
Cheers!

**Comment by barbudor** [comment]
In most of the case, yes it's just a matter of swapping
If they haven't changed the PCB and left the components connected as 'series 1', the same template would apply as is
What I suggest is after de soldering the wb3s, look for the PCB tracks. Are the GPIO from the templates seems to have tracks to the right components?

Not sure if it's worth, especially is you already have esp12s in hand, but there are esp32c3 modules compatible with Esp12 pinout
That could bring better pwm, berry language, IPv6, future support of matter protocol...
Just to let you know
